src/Components/Pages/TransferPage.tsx

```diff
@@ -377,15 +377,18 @@ const TransferPage = () => {
             disabled: !homeConfig || !address,
           })}
         >
+          {console.log(destinationChainConfig)}
           <section>
             <SelectInput
               label="Destination Network"
               className={classes.generalInput}
               disabled={!homeConfig}
-              options={destinationChains.map((dc) => ({
-                label: dc.name,
-                value: dc.chainId,
-              }))}
+              options={destinationChains
+                .filter((dc) => dc.chainId === 1)
```

Comment:
We can not be sure that chainId will be the same on all envs. We should think about more flexible solution.

Reply:
No, In all three env the chainId for cere network is set to 1. 
Even in future if we need to differentiate the network (Testnet, Devnet, Mainnet), we can add one more param as NetworkId like Ethereum.